sim/priest/shadow/spriest.go

```diff
@@ -253,7 +255,11 @@ func (spriest *ShadowPriest) Act(sim *core.Simulation) time.Duration {
 		// what do you even do... i guess just sit around
 		mbcd := spriest.Character.GetRemainingCD(priest.MBCooldownID, sim.CurrentTime)
 		swdcd := spriest.Character.GetRemainingCD(priest.SWDCooldownID, sim.CurrentTime)
-		wait = core.MinDuration(mbcd, swdcd)
+		vtidx := spriest.VTSpell.DotInput.TimeRemaining(sim) - vtCastTime
```

Comment:
This case is only hit if you have every spell on CD and don't have mindflay talent... which probably should never happen.